OpenEXR/config/OpenEXRConfig.cmake.in

```diff
@@ -0,0 +1,9 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(Threads)
```

Comment:
Is this sufficient, or should this dependency be specified with an @SOMETHING@ set from IlmThreads? Just wondering because if we use std::thread on windows as an example, we don't need to additionally specify system level threading libraries to link to.

Reply:
This method certainly works for me and doesn't seem to have any Windows conditional things in the build system itself.

https://github.com/AcademySoftwareFoundation/openexr/blob/afa84f87ae24546f71a0e9ffac400e92a8da8b99/IlmBase/IlmThread/CMakeLists.txt#L30
https://github.com/AcademySoftwareFoundation/openexr/blob/afa84f87ae24546f71a0e9ffac400e92a8da8b99/IlmBase/config/IlmBaseSetup.cmake#L133

I have added REQUIRED to those packages, as they are.

Reply:
sounds good.